sklearn/preprocessing/label.py

```diff
@@ -413,6 +416,10 @@ def label_binarize(y, classes, neg_label=0, pos_label=1, sparse_output=False):
     sparse_output : boolean (default: False),
         Set to true if output binary array is desired in CSR sparse format
 
+    force_matrix : boolean (default: False),
+        When true, binary targets are transformed to a 2d array;
```

Comment:
I think the output is 2d in any case. The difference is whether there are two columns or 1.